Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md

```diff
@@ -21,13 +21,13 @@ The NuGet Package Manager can be used to search and install NuGet packages in th
 
 1.	Right-click the project or solution in the Solution Explorer tab, and choose **Manage NuGet Packages…** 
 
-    ![NuGet package manager add-in for Windows](NuGet_Packages_Images/img6.png)
+    ![NuGet package Extension manager add-in for Windows](extension-syncfusion-nuget-packages_images/nuget-package-extension-window.png)
```

Comment:
Dont change the previous image folder name. If the Image Folder name has capital letters, you should change to small case, otherwise don't change the previous image folder name.

Check and correct the changes in all places.

Reply:
Have made the changes.